.changeset/smooth-scissors-speak.md

```diff
@@ -3,3 +3,5 @@
 ---
 
 Added min-w-0 to the button component to enable truncation for components that use button.
+
+https://css-tricks.com/flexbox-truncated-text
```

Comment:
I meant as a code comment so readers of the code know why `min-width` is there 😆.